src/app/frontend/secret/list/card.html

```diff
@@ -32,6 +32,12 @@
         </kd-middle-ellipsis>
       </div>
     </kd-resource-card-column>
+    <kd-resource-card-column ng-if="$ctrl.secret.type">
```

Comment:
ok! like  this
 <kd-resource-card-column>
      <kd-middle-ellipsis display-string="{{$ctrl.secret.type}}">
      </kd-middle-ellipsis>
      <div ng-hide="::$ctrl.secret.type">-</div>
    </kd-resource-card-column>